psl-groovy/src/main/java/org/linqs/psl/groovy/PSLModel.groovy

```diff
@@ -53,6 +53,7 @@ public class PSLModel extends Model {
 	// Keys for Groovy syntactic sugar
 	private static final String predicateKey = 'predicate';
 	private static final String predicateArgsKey = 'types';
+	private static final String predicateArgNamesKey = 'names';
```

Comment:
I don't really like the key "names", it is too vague.
I am on the fence about "types", but at least that cannot refer to the predicate itself.